app/views/devise/shared/_links.html.erb

```diff
@@ -16,6 +16,7 @@
       <%= link_to "Sign in with Google", omniauth_authorize_path(resource_name, provider) %><br />
     <%- else %>
       <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br />
+      <%= link_to "Sign in with Google", omniauth_authorize_path(resource_name, provider) %><br />
```

Comment:
why do you have google sign in link here?

Reply:
without this, there will be only "Sign in with Facebook"
![image](https://user-images.githubusercontent.com/55602048/136051210-1cc4ecd8-200f-4376-9f7f-974d4a336c4d.png)

Reply:
yeah, because you made a mistake in User model